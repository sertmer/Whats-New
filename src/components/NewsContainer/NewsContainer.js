import React from 'react';
import './NewsContainer.css'
import NewsArticle from '../NewsArticle/NewsArticle';

const NewsContainer = ({data}) => {
  let articles = data.map(story => {
  let {id, headline, description, img, url} = story
    return (
      <NewsArticle 
        key={id} 
        headline={headline} 
        description={description}
        img={img}
        url={url}
      />
    )
  })
  return (
    <section className='article-container'>
      {articles}
    </section>
  )
}

export default NewsContainer;