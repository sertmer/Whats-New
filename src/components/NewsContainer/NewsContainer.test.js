import React from 'react'
import NewsContainer from './NewsContainer'
import { shallow } from 'enzyme'

describe('NewsContainer', () => {
  it('should match the snapshot with all data passed in correctly', () => {
    const wrapper = shallow(<NewsContainer 
      data={[{id:1}]}
      />)
    expect(wrapper).toMatchSnapshot()
  })
})