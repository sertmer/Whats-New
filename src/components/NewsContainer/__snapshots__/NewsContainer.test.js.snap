// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewsContainer should match the snapshot with all data passed in correctly 1`] = `ShallowWrapper {}`;
